chore(na): add 5.0 release notes (#2054)
medic/cht-docs: content/en/releases/5_0_0.md

## Upgrade notes

### Breaking changes

As mentioned above, be sure to review the [upgrade documentation](/hosting/cht/migration/preparing-for-5/) before upgrading to appreciate what the following changes mean for your deployment:

- [#10183](https://github.com/medic/cht-core/issues/10183): Replication fails to filter out reports with needs_signoff set to false
- [#10429](https://github.com/medic/cht-core/issues/10429): Require Chrome 107 to support Angular uplift
- [#5906](https://github.com/medic/cht-core/issues/5906): Make declarative config mandatory
- [#8157](https://github.com/medic/cht-core/issues/8157): Deprecate enabling languages through generated docs
- [#9202](https://github.com/medic/cht-core/issues/9202): Increase ecmaversion linting for ddocs
- [#9542](https://github.com/medic/cht-core/issues/9542): Reduce disk space with CouchDB Nouveau (TCO v1)
- [#9954](https://github.com/medic/cht-core/issues/9954): Hide upgrade button in admin app for k8s deployments, while still allowing staging upgrades
- [#9983](https://github.com/medic/cht-core/issues/9983): Require `app_url` to be set when enabling `token_login`

### UI/UX changes

> None.

Author: I don't think it is.  It's using an existing feature (training cards). Further, it just presents a modal to the very most technical of users, CHT Admins.

####  Seconds to synchronize (lower is better)

<!--
See source for chart:
https://docs.google.com/spreadsheets/d/1RBnfmH0QApvof3XXHLM2DHAtVuwYcKxvlgYvA57LCpM 
-->
<img class="graph-key" src="../images/5_0_4color.png" style="width:20px; " title=" CHT 4.21"> CHT 4.21
<img class="graph-key" src="../images/5_0_5color.png" style="width:20px; " title=" CHT 5.0"> CHT 5.0
{{< cards cols="1">}}
{{< card  image="../images/5_0_faster_replication.png"   title="Up to 69% faster synchronization" subtitle="Chart showing users with 2.3k,  83k, 126.2k, 110.9k and 121.8k documents synchronizing against CHT 4.21 (blue) and CHT 5.0 (red)" >}}
{{< /cards >}}
<style>
.graph-key {
    display: inline-block;
}
.content :where(.graph-key):not(:where([class~=not-prose],[class~=not-prose] *)) {
  margin-top: 0;
  margin-bottom: 0;
}
</style>

- [#10262](https://github.com/medic/cht-core/issues/10262): Improve replication performance by moving docs_by_replication_key to nouveau

### Welcome prompt when starting a new instance for how to easily add data

> Administrators logging into the CHT for the first time often complained of being confused if the app was working or not.  They would see an error like message saying `No message selected`.  CHT 5.0 leverages [existing training card](/building/training/training-cards-resources/) technology to help direct administrators on how to get started with adding contacts and configuring the app. This includes helpful tips:

Reviewer: ```suggestion
Administrators logging in to the CHT for the first time reported confusion about whether the app was working. They would see an error message saying `No message selected`.  CHT 5.0 leverages [existing training card](/building/training/training-cards-resources/) technology to help administrators get started with adding contacts and configuring the app. This includes helpful tips:
```
